.github/workflows/ci.yml

```diff
@@ -0,0 +1,131 @@
+name: CI
+
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  testsuite-linux:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.4']
```

Comment:
```suggestion
        php-version: ['7.2', '8 0']
```

Reply:
I don't know if we've done the work to get migrations working on 8.0. I can add that to the matrix once tests are passing again.